COLLABORATOR_GUIDE.md

```diff
@@ -96,13 +96,28 @@ The CTC should serve as the final arbiter where required.
 
 ## Landing Pull Requests
 
+
+* Please never use GitHub's green ["Merge Pull Request"](https://help.github.com/articles/merging-a-pull-request/#merging-a-pull-request-using-the-github-web-interface) button.
+  * If you do, please force-push removing the merge.
+  * Reasons for not using the web interface button:
+    * The merge method will add an unnecessary merge commit.
+    * The rebase & merge method adds metadata to the commit title.
+    * The rebase method changes the author.
+    * The squash & merge method has been known to add metadata to the commit title.
+    * If more than one author has contributed to the PR, only the latest author will be considered during the squashing.
+    
 Always modify the original commit message to include additional meta
 information regarding the change process:
 
 - A `Reviewed-By: Name <email>` line for yourself and any
   other Collaborators who have reviewed the change.
   - Useful for @mentions / contact list if something goes wrong in the PR.
   - Protects against the assumption that GitHub will be around forever.
+
+  * Easiest to use `git log`, then do a search.
+  * In vim: `/Name` + `enter` (+ `n` as much as you need to)
```

Comment:
I think you mean the "default git pager, `less`" here, not `vim`? Usually `git log`, when output is longer than the terminal uses the git pager, which defaults to `less`, and `less` (as well as the old pager, `more`, `vim`, and lots of tools) uses `/` for searching.